obsbot/cogs/public/log_analysis.py

```diff
@@ -109,27 +118,33 @@ async def on_message(self, msg: Message):
                     continue
                 except ClientResponseError:  # file download failed
                     logger.error(f'Failed retrieving log from "{raw_url}"')
+                    react('❌')
```

Comment:
Could just write it with async code as  `await react_with('❌')`

Using async instead of creating a new task would also make sure that emojis are delivered to the target message in-order.

The underlying `on_message` callback runs in its own async task, so awaiting things within it is not holding up any other messages, if that was the purpose behind creating new tasks for emoji reactions.